src/platform/plugins/private/links/public/components/dashboard_link/dashboard_link_component.tsx

```diff
@@ -151,10 +160,11 @@ export const DashboardLinkComponent = ({ link, layout, parentApi }: DashboardLin
   return (
     <EuiListGroupItem
       size="s"
+      wrapText={textOverflow === LINK_TEXT_OVERFLOW_WRAP}
```

Comment:
The default should be `ellipses` so that existing Links panels (which do **not** have this setting defined) do not get modified